packages/backend/src/index.ts

```diff
@@ -63,6 +64,7 @@ async function main() {
   app.use(requestLoggingHandler());
   app.use('/catalog', await catalog(createEnv('catalog')));
   app.use('/scaffolder', await scaffolder(createEnv('scaffolder')));
+  app.use('/auth', await auth(createEnv('auth')));
```

Comment:
right now it'll create a database and never use it, but no biggie, we can clean the init stuff up separately :)